Userland/Libraries/LibWeb/HTML/Scripting/WorkerEnvironmentSettingsObject.cpp

```diff
@@ -13,9 +13,10 @@ namespace Web::HTML {
 JS_DEFINE_ALLOCATOR(WorkerEnvironmentSettingsObject);
 
 // https://html.spec.whatwg.org/multipage/workers.html#set-up-a-worker-environment-settings-object
-JS::NonnullGCPtr<WorkerEnvironmentSettingsObject> WorkerEnvironmentSettingsObject::setup(JS::NonnullGCPtr<Page> page, NonnullOwnPtr<JS::ExecutionContext> execution_context /* FIXME: null or an environment reservedEnvironment, a URL topLevelCreationURL, and an origin topLevelOrigin */)
+JS::NonnullGCPtr<WorkerEnvironmentSettingsObject> WorkerEnvironmentSettingsObject::setup(JS::NonnullGCPtr<Page> page, NonnullOwnPtr<JS::ExecutionContext> execution_context, SerializedEnvironmentSettingsObject const& outside_settings, WorkerGlobalScope::WorkerType worker_type)
```

Comment:
The spec link shows that we're supposed to pass a `number unsafeWorkerCreationTime`. Can we pass that as an unused double argument with a `(void)unsafe_worker_creation_time;` at the top of the method before step 1?